subsys/bluetooth/host/smp.c

```diff
@@ -4569,6 +4588,11 @@ static void bt_smp_connected(struct bt_l2cap_chan *chan)
 	       CONTAINER_OF(chan, struct bt_l2cap_le_chan, chan)->tx.cid);
 
 	k_work_init_delayable(&smp->work, smp_timeout);
+#if defined(CONFIG_BT_SETTINGS) && defined(CONFIG_BT_GATT_CLIENT)
+	k_work_init(&smp->work_settings_ccc_and_cf_save,
```

Comment:
Here you're not using the parenthesis, so you're also being inconsistent with yourself :)